### DISCUSSION TITLE
TUTORIEL OU MODE EMPLOI A320 A32NX

### DISCUSSION THREAD

**Discussion by kampinod** [flybywiresim/aircraft#3741] [General]
Bonjour, j'utilise régulièrement A32NX FBW depuis plusieurs mois, mais bien que j'applique les consignes trouvées sur les tutoriels consacrés au sujet, et sauf mauvaise manip de ma part, le Pilote Automatique (Altitude et V/S), ne répondent pas comme ils devraient...Par ailleurs l'avion ne semble pas respecter les consignes d'Altitude du MCDU, sans parler du fait que les distances restantes annoncées dans la page F-PLN ne sont pas cohérentes avec les différentes branches du PLN, ce qui rend très difficile de calculer le Top Of Descent, mais peut-être qu'une partie de ces problèmes ne sont pas de votre fait, mais de MSFS...?
**Pour conclure, existe-t-il un tuto ou un mode d'emploi fiable? Si non, serait-il possible de le mettre en place pour que nous puissions profiter pleinement de ce bel avion...**
Je suis malgré tout admiratif de l'excellent et énorme travail fourni par la Communauté et je vous en remercie sincèrement...

**Comment by hotshotp** [comment]
Flight plan manager and autopilot is currently being reworked

**Reply by kampinod** [reply]
Merci pour votre réponse et bon courage pour votre travail...
Par ailleurs savez si une méthodologie pour l'utilisation du PA et du MCDU sera prévue pour les utilisateurs...?

**Comment by kampinod** [comment]
Bonjour, j'utilise tous les jours FS et plus particulièrement A32NX FBW, et je voudrais vous signaler une anomalie, j'ai constaté que (sauf coincidence!) lorsqu'on utilise la tablette, et notamment le calcul du TOD (Performance), cela semble perturber l'autopilote, car l'avion ne suit plus le plan de vol prévu, même lorsqu'on désactive, puis réactive l'autopilote, en désespoir de cause, j'utilise le HDG sou AP pour continuer mon vol, mais l'avion a un peu de mal à suivre le cap assigné par le HDG, il y a un écart de 4 à 5° entre le cap suivi par l'avion et le cap du HDG...

**Reply by kampinod** [reply]
J'ai omis de signaler, que lorsque j'ai éteint la tablette, (après utilisation Calcul du TOD) l'avion a repris le cap assigné par le plan de vol...
